README.md

```diff
@@ -166,3 +166,18 @@ You can skip the whole `node_modules` by adding to `tsconfig.json`:
   }
 }
 ```
+
+### Checking the type of the return value
+
+To check the return value of a library, simply set the type to the variable in your `index.d.ts`:
+
+```ts
+const test: string = await myFunction();
+```
+
+If the function only returns a string a never a number, then this will fail:
+
+```ts
+// FAILS
```

Comment:
We use different control comment for errors. And we need to add error text to it (to be sure that it is exactly the conflict with return type